recipes/netcdf/all/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "4.7.4":
+    url: "https://github.com/Unidata/netcdf-c/archive/v4.7.4.tar.gz"
+    sha256: "99930ad7b3c4c1a8e8831fb061cb02b2170fc8e5ccaeda733bd99c3b9d31666b"
+  "4.7.3":
```

Comment:
Do you really need all those old versions? Why not distributing only the latest? It should be safer and more stable I think.

Reply:
I was not sure if having many versions is desirable or not. But as I rethink about it, it doesn't seem to be useful to have multiple (especially minor) versions of the library.